### PR TITLE
Merge 2.9 -> develop

### PR DESCRIPTION
**Pull request by SimonRichardson**
Merge 2.9 into develop:

Conflicts was just invalid imports.

822fc3541b (upstream/2.9, origin/2.9, 2.9) Merge pull request #12561 from SimonRichardson/charmhub-double-refresh

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,6 +17,7 @@ require (
 	github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac
 	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
 	github.com/coreos/go-systemd/v22 v22.0.0-20200316104309-cb8b64719ae3
+	github.com/davecgh/go-spew v1.1.1
```
Comment (achilleasa) [inline go.mod:20]: What is using this?

Comment (SimonRichardson) [reply go.mod:20]: nothing, this should be a lint failure.

**Review by achilleasa — APPROVED** [review]
LGTM

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**